openapi/components/schemas/news/statusDate.yaml

```diff
@@ -2,4 +2,7 @@ allOf:
   - description: |
       When did the article's status change to its current
       value?
-  - $ref: ../dateTime.yaml
+      
+      This read-only object representation is deprecated.  It will be
+      replaced in 23.1 with an RFC3339 string eg. "2018-01-28T00:00:00+11:00".
```

Comment:
I thought this is for the master branch?

Reply:
Maybe I misunderstood.

I created the same PR for both develop and master, because both are now at version 23.0.

Should the develop branch have been for 23.1?  (So documenting the date string representation that we plan to introduce in 23.1)

Reply:
> Should the develop branch have been for 23.1? (So documenting the date string representation that we plan to introduce in 23.1)

I think so. The master branch is now for 23.0 and the develop branch should be for 23.1. We have to switch the master branch as we did for the product.

Reply:
Okay, I will update this schoolbox-api-docs develop PR to target 23.1, and create a PR also for schoolbox develop to make the actual change to the API.

Reply:
I'm planning to keep this repo's versions in sync with the base schoolbox repo: that means that the (current) develop PRs for this repo will be applicable once the docs for version 23.1 go live.